### PR TITLE
Iconography: essencials

### PR DESCRIPTION
**Pull request by Michele-Camargo**
# What

Implement the new icons proposed in the [Iconography](https://www.figma.com/file/9WVVk8Vobz2ZMzmHjtEFEh/Iconography?node-id=1825%3A1490) spec

# Why

Enable us to have a greater visual collection, once we have a greater and more diverse amount of icons

# How

Added informations components in /components/Icons

# Sample

https://user-images.githubusercontent.com/44146877/167705960-20b91b72-fd3c-4b26-8a55-f5b20c38305f.mov

# QA

### CODE REVIEW

**Review by LopesAlysson — APPROVED** [review]

**Review by devjonathansouzasi — APPROVED** [review]
Show 🚀

File: src/components/Icons/Essencials/CalendarDataRange.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function CalendarData({
+  id = 'CalendarDataIcon',
```
Comment (albertozaranza) [inline src/components/Icons/Essencials/CalendarDataRange.tsx:10]:
```suggestion
function CalendarDataRange({
  id = 'CalendarDataRangeIcon',
```

File: src/components/Icons/Essencials/Flag.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  height = getSize(),
+  ...props
+}: IconProps) {
+  const fill = React.useMemo(() => getFill({ gradient: props.gradient, color, id }), [color, props.gradient, id]);
+
+  return (
+    <BaseIcon id={id} color={color} width={width} height={height} viewBox={viewBox} {...props}>
+      <Path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (lucianomlima) [inline src/components/Icons/Essencials/Flag.tsx:23]: 🔧 Passou batido esses aqui
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

**Review by lucianomlima — APPROVED** [review]
Só um que ficou sem ajeitar as props 😉

File: src/components/Icons/Essencials/Flag.tsx (hunk 2 above)
Comment (Michele-Camargo) [reply src/components/Icons/Essencials/Flag.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/574/commits/897149847831d1ec7f69709265117ecc9502761a

File: src/components/Icons/Essencials/CalendarDataRange.tsx (hunk 1 above)
Comment (Michele-Camargo) [reply src/components/Icons/Essencials/CalendarDataRange.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/574/commits/897149847831d1ec7f69709265117ecc9502761a

**Pull request merged**